src/SignalR/clients/cpp/src/signalrclient/Build/VS/signalrclient.vcxproj

```diff
@@ -17,16 +17,16 @@
   <Import Project="..\..\..\..\Build\Config.Definitions.props" />
```

Comment:
Can you do me a big favor and make WindowsTargetPlatformVersion = 10.0.17134.0 everywhere (including here)? This PR I merged from 2.1 did some but I forgot to update more when merging from 2.1 to 2.2. https://github.com/aspnet/AspNetCore/commit/80bb14c3ef423cea4b970cc69ff273056c277428

Reply:
You should just be able to find replace for 10.0.15063.0